src/pages/trade-types/cfds/_start-trading.tsx

```diff
@@ -16,40 +16,34 @@ const StartTrading = () => {
                 <SmallContainer direction="column" ai="flex-start">
                     {is_mobile_or_tablet ? (
                         <Header as="h3" type="section-title" mb="4rem">
-                            {localize('Start trading CFDs on Deriv')}
+                            <Localize translate_text="_t_Start trading CFDs on Deriv_t_" />
                         </Header>
                     ) : (
                         <Header as="h3" type="section-title" mb="4rem">
                             <Localize
-                                translate_text="Start trading CFDs on <0/>Deriv<0/>"
+                                translate_text="_t_Start trading CFDs on <0/>Deriv<0/>_t_"
                                 components={[<br key={0} />]}
                             />
                         </Header>
                     )}
                     <Timeline>
-                        <Timeline.Item title={<Localize translate_text="Practise" />}>
-                            {is_eu
-                                ? localize(
-                                      'Open a demo CFDs account and practise with an unlimited amount of virtual funds.',
-                                  )
-                                : localize(
-                                      'Open a demo Deriv MT5 or demo Deriv X account and practise with an unlimited amount of virtual funds.',
-                                  )}
-                        </Timeline.Item>
-                        <Timeline.Item title={<Localize translate_text="Trade" />}>
-                            {is_eu
-                                ? localize(
-                                      'Trade with a real CFDs account. Get access to leverage and trade positions larger than your existing capital.',
-                                  )
-                                : localize(
-                                      'Trade with a real Deriv MT5 or real Deriv X account and get access to high leverage to trade positions larger than your existing capital.',
-                                  )}
+                        <Timeline.Item title="_t_Practise_t_">
+                            {is_eu ? (
```

Comment:
`const sth = is_eu ? "_t_Open a demo CFDs account and practise with an unlimited amount of virtual funds._t_" : "_t_Open a demo Deriv MT5 or demo Deriv X account and practise with an unlimited amount of virtual funds._t_"`

```
<Localize translate_text={sth} />
```